aep/general/0202/aep.md.j2

```diff
@@ -0,0 +1,52 @@
+# Fields
+
+The [`buf.validate.field`][protovalidate] type, through its accompanying
+extension `buf.validate.field`, enriches a field's schema beyond the basic
+name and type information.
+
+## Guidance
+
+Decorating a field with `buf.validate.field` is only necessary when a field
+consistently follows a set pattern.
+
+#### Format Compatibility
+
+Adding a format specifier to an existing, unspecified field **is not** backwards
+compatible, *unless* the field in question has always conformed to the format
+being specified.
+
+Changing an existing format specifier to a different one in all cases **is not**
```

Comment:
```suggestion
Changing an existing validation rule to a different one in all cases **is not**
```

But is this true?

Simple argument:  allowing new values could break readers, and disallowing previously allowed values could break writers.  Does this hold up?